### PR TITLE
Ukrainian Translation 100% complete

### PR DESCRIPTION
**Pull request by tsunamistate**
You know what they say, the last 20% of work takes 80% of the time. For me, it was last 5%
But now, after 196 days (from initial commit on Oct 28th 2021), and 29 days from my last update, of keeping my nose in dictionaries, I present to you: 2042 out of 2042 strings translated!
And just in time for my birthday!

@SourceCodeDeleted please review when you have the time
Ignore first commit
Converting to draft until review is complete

### CODE REVIEW

**Comment by AJenbo** [comment]
Happy birthday :)

File: Translations/uk.po [hunk 1]
```diff
@@ -2388,75 +2388,75 @@ msgstr "Еліксир Живучості"
 
 #: Source/itemdat.cpp:144
 msgid "Scroll of Healing"
-msgstr "Згорток Зцілення"
+msgstr "Сувій Зцілення"
```
Comment (SourceCodeDeleted) [inline Translations/uk.po:2391]: Interesting choice. I haven't heard this word until now )I guess it lines better with Свиток

File: Translations/uk.po [hunk 2]
```diff
@@ -5870,7 +5870,7 @@ msgstr "Зв'язування Трьох"
 #. TRANSLATORS: Book Title
 #: Source/objects.cpp:276
 msgid "The Realms Beyond"
-msgstr "Царства за Межами"
+msgstr "Виміри за Межами"
```
Comment (SourceCodeDeleted) [inline Translations/uk.po:5873]: good choice. I am surprised we let the old one be.

**Review by SourceCodeDeleted — COMMENTED** [review]
10680 has a typo. I am on mobile currently and cannot mark it.

It should be :

"і після того, як вони десятиліттями сіяли хаос на сході, їх зловив проклятий "

**Comment by tsunamistate** [comment]
@SourceCodeDeleted thank you, fixed

**Comment by SourceCodeDeleted** [comment]
Ok it is good to merge @AJenbo

**Comment by AJenbo** [comment]
Nice works guys

**Pull request merged**